bundles/GraphicServiceOrder/Controller/GraphicServiceOrderController.php

```diff
@@ -62,6 +63,10 @@ public function createOrder(Request $request, OrderService $orderService)
      */
     public function createOrderSubmitted()
     {
-        return $this->render('@GraphicServiceOrderBundle/createOrderSubmitted.html.twig');
+        $order = $this->getDoctrine()->getRepository(GsOrder::class)->find($_REQUEST[0]);
```

Comment:
You should add a parameter to the controller instead, e.g.
```
…
 * @Route("/submitted/{id}", name="submitted")
 */
public function createOrderSubmitted(GsOrder $order) {
…
```

Furthermore, the action should be called something like `showOrderSubmitted` because it shows stuff.

Reply:
Changed